src/components/UniversalNavbarExpanded/constants.js

```diff
@@ -0,0 +1,82 @@
+import uuidv4 from 'uuid/v4'
+export const LINKS = {
+  // These are used e.g. in the logo, icons and bottom of mobile nav:
+  INDEX: { href: '/' },
+  TERM: {
+    href: '/term',
+    title: 'Get covered',
+  },
+  // TALK: {
+  //     id: uuidv4(),
+  //     href: '/',
+  //     title: 'Talk to us',
+  //   },
```

Comment:
Is this intentionally left here as a comment or can it be removed?

Reply:
it's gonna be part of the file once Kustomer is implemented on CMS

Reply:
marked in TODOs for fastfollow